swagger.yaml

```diff
@@ -0,0 +1,249 @@
+swagger: "2.0"
+
+info:
+  title: "Tweet API"
+  description: "This provides an API for creating a message (post), reading a message (get), editing a message (put), delete a message (delete)."
+  version: "1.0"
+
+host: localhost
+
+tags:
+- name: "tweets"
+
+schemes:
+  - "https"
+
+consumes:
+  - application/json
+produces:
+  - application/json
+
+paths:
+
+  /tweets:
+  
+    get:
+      tags:
+      - tweets
+      description: "search for a user's tweets"
+      produces:
+      - application/json
+
+      parameters:
+      - in: "query"
+        name: userName
+        required: false
+        type: string
+
+      - in: "query"
+        name: tweetId
+        required: false
+        type: integer
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
```

Comment:
Will this API only return a single tweet every time? What will I get if I search with `tweetId` and `userName` are not matched? For example, if the `userName` of `tweetId=1` is `wutso`, what will I get if I send a request to `GET /tweets?userName=alawammo`?

Reply:
searching with userName should return all the tweets for that user, 
searching with tweetId should return only one tweet

Reply:
So you probably wanna redefine the returned schema here since it looks like you will only return a single object right now.